pages/dao/[symbol]/proposal/components/instructions/Vsr/VotingMintConfig.tsx

```diff
@@ -28,18 +27,25 @@ import {
   Registrar,
 } from 'VoteStakeRegistry/sdk/accounts'
 import { DEFAULT_VSR_ID, VsrClient } from 'VoteStakeRegistry/sdk/client'
+import { HeliumVsrClient } from 'HeliumVotePlugin/sdk/client'
 import useWallet from '@hooks/useWallet'
+import { heliumVsrPluginsPks, vsrPluginsPks } from '@hooks/useVotingPlugins'
 
-interface ConfigureCollectionForm {
+interface ConfigureVotingMintForm {
   programId: string | undefined
   governedAccount: AssetAccount | undefined
   mint: string
   mintIndex: number
-  grantAuthority: AssetAccount | undefined
   mintDigitShift: number
-  mintUnlockedFactor: number
-  mintLockupFactor: number
-  mintLockupSaturation: number
+  maxLockupFactor: number
+  lockupSaturation: number
+
+  // Helium DynamicFields based on programId
+  baselineVoteWeightFactor?: number
```

Comment:
i would like it if these all lived grouped together in one object. 
so `helium.baselineVoteWeightFactor` instead of `baselineVoteWeightFactor`. This would also let you be more typesafe.

Reply:
actually, thanks for calling this out. In our vsr we used to require a `minimumLockupUpTs` but now we dont and baselineVoteWeightFactor is shared across both vsr instances. The only actual logic happening here is hiding/showing grantAuth if the programId isnt helium. Don't even need these fields anymore